cli/src/config.ts

```diff
@@ -239,9 +239,6 @@ async function loadAndroidConfig(
   const buildOptions = {
     keystorePath: extConfig.android?.buildOptions?.keystorePath,
     keystorePassword: extConfig.android?.buildOptions?.keystorePassword,
-    keystoreAlias: extConfig.android?.buildOptions?.keystoreAlias,
-    keystoreAliasPassword:
-      extConfig.android?.buildOptions?.keystoreAliasPassword,
```

Comment:
It's mistake, return pls))

Reply:
Yeah, so I created a pull request to correct it, but still not merged yet. 😞